fiat-core/src/main/java/com/netflix/spinnaker/fiat/model/UserPermission.java

```diff
@@ -86,13 +87,15 @@ public View getView() {
   }
 
   @Data
+  @EqualsAndHashCode(callSuper = false)
```

Comment:
I added these to get ride of the compile warnings. Since the base class is empty, it should have no effect. Equals between sets is defined in the javadocs as:


> boolean equals(Object o)
> Compares the specified object with this set for equality. Returns true if the specified object is also a set, the two sets have the same size, and every member of the specified set is contained in this set (or equivalently, every member of this set is contained in the specified set). This definition ensures that the equals method works properly across different implementations of the set interface.